services/get_account_details.proto

```diff
@@ -0,0 +1,242 @@
+
+import "timestamp.proto";
+import "duration.proto";
+import "basic_types.proto";
+import "query_header.proto";
+import "response_header.proto";
+import "crypto_add_live_hash.proto";
+import "google/protobuf/wrappers.proto";
+
+/**
+ * Get all the information about an account, including balance and allowances. This does not get the list of
+ * account records.
+ */
+message GetAccountDetailsQuery {
+    /**
+     * Account details sent from client to node, including the signed payment, and what kind of
+     * response is requested (cost, state proof, both, or neither).
+     */
+    QueryHeader header = 1;
+
+    /**
+     * The account ID for which information is requested
+     */
+    AccountID account_id = 2;
+}
+
+/**
+ * Response when the client sends the node GetAccountDetailsQuery
+ */
+message GetAccountDetailsResponse {
+    /**
+     * Standard response from node to client, including the requested fields: cost, or state proof,
+     * or both, or neither
+     */
+    ResponseHeader header = 1;
+
+    message AccountDetails {
+        /**
+         * The account ID for which this information applies
+         */
+        AccountID account_id = 1;
+
+        /**
+         * The Contract Account ID comprising of both the contract instance and the cryptocurrency
+         * account owned by the contract instance, in the format used by Solidity
+         */
+        string contract_account_id = 2;
+
+        /**
+         * If true, then this account has been deleted, it will disappear when it expires, and all
+         * transactions for it will fail except the transaction to extend its expiration date
+         */
+        bool deleted = 3;
+
+        /**
+         * The Account ID of the account to which this is proxy staked. If proxyAccountID is null,
+         * or is an invalid account, or is an account that isn't a node, then this account is
+         * automatically proxy staked to a node chosen by the network, but without earning payments.
+         * If the proxyAccountID account refuses to accept proxy staking , or if it is not currently
+         * running a node, then it will behave as if proxyAccountID was null.
+         */
+        AccountID proxy_account_id = 4;
+
+        /**
+         * The total number of tinybars proxy staked to this account
+         */
+        int64 proxy_received = 6;
+
+        /**
+         * The key for the account, which must sign in order to transfer out, or to modify the
+         * account in any way other than extending its expiration date.
+         */
+        Key key = 7;
+
+        /**
+         * The current balance of account in tinybars
+         */
+        uint64 balance = 8;
+
+        /**
+         * [Deprecated]. The threshold amount, in tinybars, at which a record is created of any
+         * transaction that decreases the balance of this account by more than the threshold
+         */
+        uint64 generate_send_record_threshold = 9 [deprecated=true];
```

Comment:
We don't even have these thresholds in state any more; so probably no reason include them in this message. 😁

Reply:
Removed them . Thanks Michael !